fundamentals/script.js

```diff
@@ -0,0 +1,274 @@
+const colorInput = document.querySelector('.color-selector');
+const widthInput = document.querySelector('.width-slider');
+const heightInput = document.querySelector('.height-slider');
+const colorBlock = document.querySelector('.color-block');
+const blockControlsWrapper = document.querySelector('.block-controls-wrapper');
+
+const TEST_DATA = {
+  ARRAY_OF_INTS: [1, 5, 6, 7]
+}
+
+var student = {
+  NAME : "David Rayy",
+  SCLAS : "VI",
+  ROLLNO : 12
+};
+
+/**
+ * TASK 1
+ *
+ */
+function getBiggerNumber(int1, int2) {
+  if ((int1 >= 40 && int1 <= 60) && (int2 >= 40 && int2 <= 60)) {
+    return int1 > int2 ? int1 : int2;
+  }
+
+  return 0;
+}
+
+/**
+ * TASK 2
+ *
+ */
+function getArraySum(intArr) {
+  return intArr.reduce(function (accumulator, currentValue) {
+    return accumulator += currentValue;
+  }, 0);
+}
+
+/**
+ * TASK 3
+ *
+ */
+function colorMixer() {
+  myColor = ["Red", "Green", "White", "Black"];
+  let output = myColor.join(',');
+}
+
+/**
+ * TASK 4
+ *
+ */
+function listProperties() {
+  for (let key in student) {
+    if (student.hasOwnProperty(key)) {
+      returnString += key + ',';
+    }
+  }
+}
+
+var stringToFormat = ' asd asd jjj bbb jjj '; // Global on purpous
```

Comment:
It's spelt "purpose". Global variable declarations usually sit on top of the scope they're declared in

Reply:
Yes sometimes I have a problem with spelling but i'm working on it.